.github/workflows/editor-changelog.yaml

```diff
@@ -0,0 +1,167 @@
+name: Generate Editor Changelog
```

Comment:
Interesting code. 
Have you searched for actions out there that do the same? Because, if there is any, we wouldn't have to maintain this code here.